Source/Libraries/GSF.Web/Model/ModelController.cs

```diff
@@ -162,29 +162,22 @@ public ModelController()
         [HttpGet, Route("New")]
         public virtual IHttpActionResult GetNew()
```

Comment:
It looks to me like you removed the try-catch from every function except this one.